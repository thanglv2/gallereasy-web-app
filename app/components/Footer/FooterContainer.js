
import styled from 'styled-components';

const FooterContainer = styled.footer`
  display: flex;
  justify-content: space-between;
  padding: 15px 50px;
  background-color: ${props => props.theme.darkgrey};
  span {
    color: ${props => props.theme.dimgrey};
    font-weight: 500;
  }
`;

export default FooterContainer;