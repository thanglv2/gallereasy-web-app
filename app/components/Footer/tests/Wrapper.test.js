import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'

import FooterContainer from '../Wrapper';

describe('<NavigationBar />', () => {
  it('render without error', () => {
    const theme = {
        dimgrey: 'dimgrey',
        darkgrey: 'darkgrey'
    }
    const tree = renderer.create(<FooterContainer theme={theme} />).toJSON()
    expect(tree).toMatchSnapshot()
  });
});
