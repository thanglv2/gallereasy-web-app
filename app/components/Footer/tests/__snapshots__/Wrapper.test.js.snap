// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavigationBar /> render without error 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  position: absolute;
  height: 50px;
  bottom: 0;
  width: 100%;
  background-color: darkgrey;
}

.c0 span {
  color: dimgrey;
  font-weight: 500;
  margin: 0 50px;
}

@media (max-width:600px) {
  .c0 span {
    margin: 0 10px;
  }
}

<footer
  className="c0"
/>
`;
